libs/api/observability/src/lib/trace-wrappers/trace-decorator-trace-wrapper.ts

```diff
@@ -0,0 +1,59 @@
+import { ModulesContainer } from '@nestjs/core';
+import { InstanceWrapper } from '@nestjs/core/injector/instance-wrapper';
+
+import { SPAN_ACTIVE, TRACE_NAME } from '../decorators/trace.decorator';
+import { TraceWrapper } from './abstract-trace-wrapper';
+
+// Wraps all Methods of Providers with OpenTelemetry spans that are marked with the `Trace` decorator
```

Comment:
```suggestion
/**
 * Wraps all Methods of Providers with OpenTelemetry spans that are marked with the `Trace` decorator
 */
```

Shows the comment in docs as well.